examples/spark/images/base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:latest
+FROM java:openjdk-8-jdk
```

Comment:
(And yes, I realize we're not building Spark in this image, I'm just pegging the JDK for the runtime.)